docs/spine.md

```diff
@@ -6,15 +6,12 @@ that can be added to the `modules/` directory in your Godot source tree and buil
 with eg. `schedtool -D -e ccache scons platform=x11 tools=yes target=release_debug`
 or whatever you like.
 
-The script as referred to here is `spine-to-godot-scene.py`, which must be run from your `device/`
```

Comment:
What about something like?

"For compatibility with Godot, run `spine-to-godot-scene.py`, (henceforth referred to as "the script"), found in the [Spine to Godot](https://github.com/mjtorn/spine-to-godot/] project, from your project root (typically the `device/` directory)."

It would both emphasize that "the script" has yet to be introduced (it confused me a little), and highlight your project.